### PR TITLE
feat: Adding ccc debug mode

### PR DESCRIPTION
**Pull request by timdeluxe**

### CODE REVIEW

File: ccc/ccc.go
Comment (dploeger) [inline ccc/ccc.go:52]: I'd rather use something like "CONTINUE_ON_ERROR". "DEBUG_ccc" should IMHO print CCC debug messages instead of skipping errors.

**Review by dploeger — CHANGES_REQUESTED** [review]

File: ccc/ccc.go
Comment (timdeluxe) [reply ccc/ccc.go:52]: yes, you are right, i also thought about adding documentation. Let me first check if it works as planned and then i'll add/change both.

File: README.md.handlebars
Comment (dploeger) [inline README.md.handlebars:235]:
```suggestion
Use the `docker logs` command with the failed container to see the complete log output. You can enhance the log by using the "DEBUG_<feature>" options. If you are really stuck, you can convince the container to keep running by setting
```

File: ccc/ccc.go
Comment (dploeger) [inline ccc/ccc.go:53]: This should be indented one level more.

File: feature/velero/install.sh
Comment (dploeger) [inline feature/velero/install.sh:6]: This would be a BREAKING_CHANGE because it require a change in configuration for users

Comment (dploeger) [reply feature/velero/install.sh:6]: IMHO I would remove this, because while I can understand the idea here, it introduces a difference to the referenced projects versioning scheme. If a project chooses to use the "v", it's the official release name on e.g. Github and should be referenced for CloudControl.

Comment (dploeger) [reply feature/velero/install.sh:6]: And I just saw that other features apparently already do this. :/

So yes, we should decide to either go the one or the other way, but this will result in a BREAKING CHANGE.

**Review by dploeger — CHANGES_REQUESTED** [review]

File: feature/velero/install.sh
Comment (timdeluxe) [reply feature/velero/install.sh:6]: yes i know, but i find it VERY confusing using "v1.2.3" on one feature and "1.2.3" on the other feature. I would rather keep having all with just the version number to have it equal and let the feature install script take care of the details.

If that results in a breaking change, well then it does 🤷‍♂️

**Comment by timdeluxe** [comment]
note to self: TODO - function removing "v" and deprecation notice

File: README.md.handlebars
Comment (dploeger) [inline README.md.handlebars:74]:
```suggestion
strings need to be provided in semver format (f.e. 1.2.3), the feature installers will take care about prefixes for download URLs, if required.
```

Comment (dploeger) [reply README.md.handlebars:74]: (as this is not a "Github thing")

File: README.md.handlebars
Comment (dploeger) [inline README.md.handlebars:235]:
```suggestion
### When I try to start CloudControl, it keeps exiting with "./ccc: exit status 1". What can I do?
```

File: README.md.handlebars
Comment (dploeger) [inline README.md.handlebars:237]: I wonder if we need to give more info on how to get the name of the container? I use `docker ps -a` to fetch the id, but you can also use the container name, which is generated by docker compose (<current path>_cli_1).

File: README.md.handlebars
Comment (dploeger) [inline README.md.handlebars:239]: Do we need to add something like "using docker exec ..." or should it be clear to people?

File: ccc/ccc.go
Comment (dploeger) [inline ccc/ccc.go:53]: Still an indent missing here. :)

File: assets/feature-installer-utils.sh
Comment (dploeger) [inline assets/feature-installer-utils.sh:57]:
```suggestion
    echo "[DEPRECATION WARNING] Versions with a \"v\" prefix are deprecated and will be removed in CloudControl 4.0. Please only use versions without the \"v\" prefix. (Got \"${VERSION}\")" >&2
```

Comment (dploeger) [reply assets/feature-installer-utils.sh:57]: I guess, we need some sort of "deprecation alert box" in the client, because logs like these will vanish quickly. I'll take a look at this after this PR is merged.

**Review by dploeger — CHANGES_REQUESTED** [review]

File: README.md.handlebars
Comment (timdeluxe) [reply README.md.handlebars:239]: As said before, i would not explain it too far.
Or we describe it REALLY specific for a noob...
But an admin, who uses cloud control, should know how to handle containers i would say...

File: ccc/ccc.go
Comment (timdeluxe) [reply ccc/ccc.go:53]: Note to self: Don't copy examples with tabs 🤦‍♂️

File: README.md.handlebars
Comment (dploeger) [reply README.md.handlebars:239]: Allright

File: README.md.handlebars
Comment (dploeger) [reply README.md.handlebars:237]: Okay.

**Review by dploeger — APPROVED** [review]
Wonderful. Thanks. Please note the deprecation of the version in the release.

**Pull request merged**
